README.md

```diff
@@ -25,6 +25,10 @@ Add it to your project with `register` and you are done!
 const fastify = require('fastify')()
 
 fastify.register(require('fastify-mongodb'), {
+  // force to close the mongodb connection when app stopped
+  // the default value is false
+  forceClose: true,
```

Comment:
I'd like to use `false` there

Reply:
the default value is already false as described in the comment above. so generally, the example would show the non-default case.